README.md

```diff
@@ -76,3 +76,4 @@ Ballerina code is distributed under [Apache license 2.0](https://github.com/ball
 ||[postgresql](https://github.com/ballerina-platform/module-ballerinax-postgresql)| [![GitHub Release](https://img.shields.io/github/release/ballerina-platform/module-ballerinax-postgresql.svg?label=)](https://github.com/ballerina-platform/module-ballerinax-postgresql/releases)| [![Build](https://github.com/ballerina-platform/module-ballerinax-postgresql/actions/workflows/build-timestamped-master.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-postgresql/actions/workflows/build-timestamped-master.yml)| [![Trivy](https://github.com/ballerina-platform/module-ballerinax-postgresql/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-postgresql/actions/workflows/trivy-scan.yml)| [![CodeCov](https://codecov.io/gh/ballerina-platform/module-ballerinax-postgresql/branch/main/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerinax-postgresql)| [![Bugs](https://img.shields.io/github/issues-search/ballerina-platform/ballerina-standard-library?query=is%3Aopen+label%3AType%2FBug+label%3Amodule%2Fpostgresql&label=&color=brightgreen&logo=github)](https://github.com/ballerina-platform//ballerina-standard-library/issues?q=is%3Aopen+label%3AType%2FBug+label%3Amodule%2Fpostgresql)| [![GitHub pull-requests](https://img.shields.io/github/issues-pr/ballerina-platform/module-ballerinax-postgresql.svg?label=)](https://github.com/ballerina-platform/module-ballerinax-postgresql/pulls)|
+||[oracledb](https://github.com/ballerina-platform/module-ballerinax-oracledb)| [![GitHub Release](https://img.shields.io/github/release/ballerina-platform/module-ballerinax-oracledb.svg?label=)](https://github.com/ballerina-platform/module-ballerinax-oracledb/releases)| [![Build](https://github.com/ballerina-platform/module-ballerinax-oracledb/actions/workflows/build-timestamped-master.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-oracledb/actions/workflows/build-timestamped-master.yml)| [![Trivy](https://github.com/ballerina-platform/module-ballerinax-oracledb/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-oracledb/actions/workflows/trivy-scan.yml)| [![CodeCov](https://codecov.io/gh/ballerina-platform/module-ballerinax-oracledb/branch/main/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerinax-oracledb)| [![Bugs](https://img.shields.io/github/issues-search/ballerina-platform/ballerina-standard-library?query=is%3Aopen+label%3AType%2FBug+label%3Amodule%2Foracledb&label=&color=brightgreen&logo=github)](https://github.com/ballerina-platform//ballerina-standard-library/issues?q=is%3Aopen+label%3AType%2FBug+label%3Amodule%2Foracledb)| [![GitHub pull-requests](https://img.shields.io/github/issues-pr/ballerina-platform/module-ballerinax-oracledb.svg?label=)](https://github.com/ballerina-platform/module-ballerinax-oracledb/pulls)|
```

Comment:
This is autogenerated no need to edit here

Reply:
removed in https://github.com/ballerina-platform/ballerina-standard-library/pull/1929/commits/3d92859879abd09244544e8666289860f78170e5